prisma/dev-seed-utils/permits.ts

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-console */
+// Relative paths required, path aliases throw error with seed command
+import { PermitType } from '@prisma/client';
+import prisma from '../index'; // Prisma client
+import { UpsertPermit } from '../types'; // Seeding types
+
+// Seed data
+const permits = [
```

Comment:
```suggestion
const permits: Array<UpsertPermit> = [
```
This will give us type suggestions